docs/_docs/versions.md

```diff
@@ -129,10 +129,11 @@ Before performing a migration, it is important to consider all Zaps which are cu
 
 Because of that, a “breaking change” can be defined as any modification to the integration which renders existing Zaps incompatible with the new version. Examples include:
 
-* Removing an input or output field on a trigger/action
 * Changing the Authentication scheme
-* Making a previously optional input field “required”
-* Deleting a trigger/action
+* Adding a new "required" input field or making a previously optional input field “required”
+* Removing or hiding a trigger/action
```

Comment:
Hiding a trigger/action doesn't break Zaps, but it can break Zap templates. The current behavior is that ZTs stay on the old version if the new version hides a trigger/action.

So suggested change:

```suggestion
* Removing a trigger/action
```

And add this sentence somewhere appropriate:

```
Hiding a trigger/action is **not** a break change for Zaps but is for Zap templates. When you promote a new version that hides a trigger/action, Zapier will keep related Zap templates on the old version so they don't break.
```

Reply:
Thanks @eliangcs! I've edited this one, the Zap Templates section here: https://platform.zapier.com/partners/zap-templates#promoting-new-versions-of-your-integration already mentions hiding a trigger/action will break the template.